docs/content/mixin-dev-guide/distribution.md

```diff
@@ -68,4 +68,43 @@ unpublished builds from the master branch. The official Porter mixins follow
 this pattern. If you have other published tagged builds of your mixin, porter
 can handle installing them as well.
 
+## Search
+
+Porter maintains a list of remote mixins that users can search via
+`porter mixins search [NAME]`. If no name is supplied, the full listing will be
+returned.  See the help menu for all command options: `porter mixins search -h`.
+
+For example, here we search for a `cowsay` mixin:
+
+```console
+$ porter mixins search cowsay
+Name     Description                        Author           Source URL                                                    Feed URL
+cowsay   A mixin for using the cowsay cli   Porter Authors   https://github.com/deislabs/porter-cowsay/releases/download
```

Comment:
I'll update the output to, say, the `az` mixin for now.

Reply:
Thanks, I just transferred cowsay to my user since it's in a general state of brokenness. It will be a good demo anyway of a personal mixin if i ever clean it up. :)